content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -0,0 +1,172 @@
+In cases where a workload requires powerful permissions, consider the following practices:
+
+ - Limit the number of nodes running powerful pods. Ensure that any DaemonSets you run
+  are necessary and are run with least privilege.
+   nodes are least-privileged and to limit the blast radius of container escapes.
+ - Avoid running powerful pods alongside untrusted or publicly-exposed ones. Consider using 
+   [Taints and Toleration](/docs/concepts/scheduling-eviction/taint-and-toleration/), [NodeAffinity](/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity), or [PodAntiAffinity](/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity) to ensure 
+   pods don't run alongside untrusted or less-trusted Pods. Pay especial attention to
+   situations where less-trustworthy Pods are not meeting the **Restricted** Pod Security Standard.
+
+### Hardening
+
+Kubernetes defaults to providing access which may not be required in every cluster. Reviewing 
+the RBAC rights provided by default can provide opportunities for security hardening. Whilst, 
+in general, changes should not be made to rights provided to `system:` accounts some options 
+to harden cluster rights exist :- 
+
+- Review bindings for the `system:unauthenticated` group and remove where possible, as this gives 
+  access to anyone who can contact the API server at a network level.
+- Avoid the default auto-mounting of service account tokens by setting
+  `automountServiceAccountToken: false`. For more details, see
+  [using default service account token]/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server).
+  Setting this value for a Pod will overwrite the service account setting, workloads
+  which require service account tokens can still mount them.
+
+### Periodic review
+
+It is vital to periodically review the Kubernetes RBAC settings for redundant entries and 
+possible privilege escalations.
+If an attacker is able to create a user account with the same name as a deleted user,
+they can automatically inherit all the rights of the deleted user, especially the
+rights assigned to that user.
+
+## Kubernetes RBAC - privilege escalation risks 
+
+Within Kubernetes RBAC there are a number of privileges which, if granted, can allow a user or a service account
+to escalate their privileges in the cluster or affect systems outside the cluster.
+
+This section is intended to provide visibility of the areas where cluster operators 
+should take care, to ensure that they do not inadvertantly allow for more access to clusters than intended.
+
+### Listing secrets
+
+It is generally clear that allowing **Get** access on Secrets will allow a user to read their contents.
+It is also important to note that the **List** access also effectively allows for users to reveal the Secret contents.
```

Comment:
and watch

Reply:
oh yeah that makes sense. Adapted this sentence to mention watch as well :)